packages/headless/src/api/service/insight/insight-api-client.ts

```diff
@@ -78,7 +78,7 @@ export class InsightAPIClient {
 
   async query(
     req: InsightQueryRequest
-  ): Promise<InsightAPIResponse<InsightQueryResponse>> {
+  ): Promise<InsightAPIResponse<SearchResponseSuccess>> {
```

Comment:
I see that the endpoint for the insight API and the search API are different, are we certain their response are always gonna be the same?

I just want to make sure that we won't accidentally break something in the insight panel if for some reason we have to adapt the `SearchResponseSuccess` interface.